packages/tablev2/style.ts

```diff
@@ -0,0 +1,85 @@
+import * as dt from "../design-tokens/build/js/designTokens";
+import { css, cx } from "emotion";
+import { pseudoElTriangle } from "../shared/styles/styleUtils";
+
+export const headerCss = css`
+  box-sizing: border-box;
+  border-top: 1px solid ${dt.themeBorderHeavy};
+  border-bottom: 1px solid ${dt.themeBorderHeavy};
+`;
+
+export const table = (
+  gridTemplateFragments: string[],
+  firstColSticky: boolean = false
+) => css`
+  display: grid;
+  grid-template-columns: ${gridTemplateFragments.join(" ")};
+  position: relative;
+  overflow: scroll;
+  ${firstColSticky && stickyFirstCol(gridTemplateFragments.length)}
+`;
+
+const stickyFirstCol = (n: number) => `
+  .${th}:first-child, .${cell}:nth-child(${n}n + 1) {
+    position: sticky;
```

Comment:
If the table parent element does not have a height set, the header cells will not be sticky because the content will never have to scroll.